_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,329 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are instruments to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] that has a [visible][] [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed:** the `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
+- **no child changed:** the [element][html element] does not have [children][child] in the [flat tree][] whose `innerText` property also changes; and
+- **not alone:** the [element][html element] has an [ancestor][] element in the [flat tree][] with a non-empty `innerText` property whose value is different from the `innerText` of the test target.
+
+**Note:** The test targets for this rule are elements that have their text content change multiple times, on a page with further content, if that change happens without user intervention.
+
+**Note:** The 10 minute time span is an arbitrary limit which is not included in WCAG. Content that updates less frequently, or that doesn't update on a regular interval, will be inapplicable for this rule but may nonetheless fail [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. The accessibility problem tends to be less severe for longer time periods, and without a time limit, testing this rule consistently would be impractical.
+
+## Expectation 1
+
+For each test target there exists a set of [instruments][instrument] to achieve one of the following objectives:
+
+- pause and resume the change of the [visible text content][]; or
+- stop the change of the [visible text content][]; or
+- hide the content that changes; or
+- alter the frequency of the changes of the [visible text content][].
+
+**Note:** If there is more than one test target, the same [instrument][] may be used to pause, stop, hide or alter the frequency of several or even all test targets.
```

Comment:
I think you are mixing the test targets with the instruments. The test targets are the content that is auto-updating. The instruments are the control mechanisms. What this note says is that I can have one single pause button to pause all auto-updating content in the page (or I can have one pause button for each auto-updating content). You seem to be implying that the note says the same button can be used to pause and stop the auto-updates. This is not what the note says.
Please correct me if I misunderstood your comment.

Reply:
- Yes, I understood the same as you mentioned in the last sentence of your comment, that is the same button (instrument) can be used to pause, stop, and hide the auto-updates. if you meant here that "It can have one single pause button to pause all auto-updating content in the page (or I can have one pause button for each auto-updating content)." then please explain it a bit more to make it clear. because it seems that the same instrument may be used to perform all actions (pause, stop, and hide) of several or even all test targets.
 - Additional comment: Also, one more thing (maybe in assumptions) if you think is relevant and appropriate that is "it must be possible to use other contents on the page when content has been paused, stop, or hide." because it is not sufficient for content only to be paused when it is in focus (with the cursor or keyboard focus). The objective of avoiding distracting the user is not met in this case.

Reply:
- I've rephrased the note. What do you think now?

- That is okay since we don't claim to meet the SC when the rule passes. But I guess we can add another assumption to clarify that.